### PR TITLE
create an omero.figure

### PR DESCRIPTION
**Pull request by jburel**
Create a figure
one column per Mechanism of Action
The first image for each compound is selected, mean of concentration is displayed

To test:
* Run the Compound notebook

cc @pwalczysko @francesw

### CODE REVIEW

File: Compound.ipynb [hunk 1]
```diff
@@ -468,6 +470,252 @@
     "plt.savefig(home+'/figure.png')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connect to IDR\n",
```
Comment (pwalczysko) [inline Compound.ipynb:477]:
```suggestion
    "### Create OMERO.figure\n",
```

The header is the most important part, it should reflect what is the content of this section of the notebook.

File: Compound.ipynb [hunk 2]
```diff
@@ -468,6 +470,252 @@
...
+    "figure_json['panels'] = panels_json\n",
+    "figure_json[\"figureName\"] = \"Compound Mechanism of Action\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Save the figure as a json file\n",
+    "You can then download the ``.json`` file or copy its content. In OMERO.figure use ``File>Import from JSON...``."
```
Comment (pwalczysko) [inline Compound.ipynb:690]:
```suggestion
    "You can then download the ``.json`` file, by clicking ``File > Open``, selecting it and cliking ``File > Download``. Alternatively, you can copy the content of the ``json`` file directly from this notebook UI. In OMERO.figure use ``File>Import from JSON...``."
    
   
```

Comment (pwalczysko) [reply Compound.ipynb:690]: the workflow works as described, up to the point when I tried to save the figure on another OMERO.server. This failed. We should mention this fact, as the Save button is active (used the "workshop" server) in figure.

**Comment by pwalczysko** [comment]
Can we get a more colourful figure ? Color labels ? What about the headers "Inhibitor" and "Antagonist" ?
It seems two of the images are either "Inhibitor" or "Antagonist", but the third image is neither, it just does not have this label... What is the difference between an Inhibitor and Antagonist in the sense used ? They sound to me like synonyms at first look ? cc @sbesson @francesw

**Comment by jburel** [comment]
Screenshot 2021-05-12 at 09 30 26

**Comment by pwalczysko** [comment]
works as expected
ready to merge fmpov

**Pull request merged**